gateway/src/apicast/policy/buffering/init.lua

```diff
@@ -0,0 +1 @@
+return require('buffering')
```

Comment:
what about calling the policy `request_buffering` instead of `buffering` ? to be more precise?